src/index.html

```diff
@@ -174,5 +183,50 @@ <h2 class="modal__title">New Post</h2>
         </div>
       </div>
     </div>
+
+    <div class="modal" id="avatar-modal">
+      <div class="modal__container">
+        <button
+          type="button"
+          class="modal__close-btn"
+          id="#close-profile-button"
+        >
+          <img src="" alt="" />
+        </button>
+        <h2 class="modal__title">Edit avatar</h2>
+        <form id="edit-avatar" novalidate class="modal__form">
+          <label for="profile-avatar-input" class="modal__label">
+            Avatar
+            <input
```

Comment:
* Please, add `type="url"` for the `avatar` input, to validate it as a `link` https://snipboard.io/nQIwhx.jpg